views/default/page/layouts/default.php

```diff
@@ -1,36 +1,71 @@
 <?php
+
 /**
  * Elgg default layout
  *
- * @package Elgg
- * @subpackage Core
+ * @uses $vars['class']        Additional CSS classes to apply to the layout
+ *
+ * @uses $vars['breadcrumbs']  An array of breadcrumbs (see navigation/breadcrumbs)
+ * @uses $vars['nav']          Alternative page navigation (replaces navigation/breadcrumbs)
```

Comment:
Should we throw this out? If breadcrumbs is string use it, if array - render navigation/breadcrumbs?